public/javascripts/NumberGuesser.js

```diff
@@ -0,0 +1,44 @@
+class NumberGuesser {
+  constructor() {
+  this.clear()
+  }
+  attempt(guess) {
+    if(guess !== 'exit') {
+     this.guesses.push(guess, this.compare(guess))
+   }
+  }
+
+  clear() {
+    let newAnswer
+    do{
+      newAnswer = Math.ceil(Math.random() * 100)
+    }
+    while(this.answer === newAnswer)
+    this.answer = newAnswer
+    this.guesses = []
+  }
+
+  compare(guess) {
+    if(guess === 'exit') {
+      return
+    }
+    guess = parseInt(guess)
+    if(isNaN(guess)) {
+      return 'Please enter a number, no letters accepted'
```

Comment:
returning all of these long english sentences its something I've ever seen in real-world software. Does this exercise encourage or require it?